lang/cs/Org.Apache.REEF.Evaluator.Tests/ContextRuntimeTests.cs

```diff
@@ -37,13 +42,7 @@ public sealed class ContextRuntimeTests
         public void TestContextEvents()
         {
             const string hello = "Hello!";
-            var contextConfig = ContextConfiguration.ConfigurationModule
-                .Set(ContextConfiguration.Identifier, "ID")
-                .Set(ContextConfiguration.OnContextStart, GenericType<ContextEventHandler>.Class)
-                .Set(ContextConfiguration.OnContextStop, GenericType<ContextEventHandler>.Class)
-                .Set(ContextConfiguration.OnMessage, GenericType<ContextEventHandler>.Class)
-                .Build();
-            
+            var contextConfig = GetContextEventHandlerContextConfiguration();
             var injector = TangFactory.GetTang().NewInjector();
 
             var handler = new ContextEventHandler();
```

Comment:
Also for ContextEventHandler class, as it is a test class, it might be better to give a name that contains test or something to differ from prod code.